cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html

```diff
@@ -0,0 +1,235 @@
+
+{# ==========================================================================
+
+   render( data )
+
+   ==========================================================================
+
+   Description:
+
+   Renders the entire form explainer.
+
+   data:               The data structure is as follows:
+                       [
+                         { 'img': 'url-to-form-img'
+                           'terms': [ ... ]
+                         }
+                       ]
+
+   ========================================================================== #}
+
+{% macro render( data ) %}
+<div class="explain" aria-controls="form-explainer">
+    <ul class="tabs explain_tabs">
+        <li class="tab-list active-tab">
+            <button class="tab-link tab-link__checklist"
+                    data-target="checklist"
+                    tabindex="0">
+                <span class="cf-icon cf-icon-approved-round"></span>
+                <span class="tab-label">Check details</span>
+            </button>
+        </li>
+        <li class="tab-list">
+            <button class="tab-link tab-link__definitions"
+                    data-target="definitions"
+                    tabindex="0">
+                <span class="cf-icon cf-icon-help-round"></span>
+                <span class="tab-label">Get definitions</span>
+            </button>
+        </li>
+    </ul>
+    {{ render_pagination( data ) }}
+    {% for page in data %}
+    <figure class="explain_page"
+            id="explain_page-{{ loop.index }}"
+            role="tablist"
+            aria-multiselectable="true">
+        <div class="image-map explain_image-map">
+            <div class="image-map_wrapper">
+                <div class="form-explainer_page-buttons">
+                    <button class="prev a-btn a-btn__disabled">
+                        <span class="u-visually-hidden">Prev page</span>
```

Comment:
Should the text say "Previous page"? This is just for screenreaders, right? In which case it should be full words.

Reply:
Fixed.